model/task/db.go

```diff
@@ -2923,3 +2923,68 @@ func FindGeneratedTasksFromID(generatorID string) ([]GeneratedTaskInfo, error) {
 
 	return out, nil
 }
+
+type generateTasksEstimationsResults struct {
+	DisplayName        string  `bson:"_id"`
```

Comment:
You're allowed to omit a field if you don't need it in your result struct, and the driver will just ignore the `_id` that results from the group. You could just remove the field and it'll still work. Alternatively, since it looks like this aggregation only expects exactly one group to be created anyways, you could also just `$group` with `{`_id`: 1}` (or some other constant) and it'll make one group.